README.md

```diff
@@ -119,7 +119,7 @@ analysis that are not present in vanilla PostgreSQL. (For example, the `time_buc
 TimescaleDB is available pre-packaged for several platforms:
 
 - Linux:
-    - [RedHat / CentOS](https://tsdb.co/GitHubTimescaleRedHatCentOS)
+    - [RedHat](https://tsdb.co/GitHubTimescaleRedHatCentOS)
```

Comment:
Nit: Maybe we should change the name of the redirect as well (`GitHubTimescaleRedHatCentOS`)